package.json

```diff
@@ -0,0 +1,270 @@
+{
+    "name": "daskeyboard-applet--birthday-reminder",
+    "displayName": "Birthday Reminder",
+    "publisher": "Das Keyboard",
+    "version": "1.0.1",
+    "description": "You will never forget a birthday again.",
+    "longDescription": "Get birthday alerts directly on a Das Keybaord Q devices.",
+    "officialProductName": "Birthday Reminder",
+    "publisherName": "Das Keyboard",
+    "isSingleton": false,
+    "icon": "",
+    "image": "",
+    "authorName": "Das Keyboard",
+    "authorUrl": "https://twitter.com/daskeyboard",
+    "issuesUrl": "https://github.com/daskeyboard/daskeyboard-applet--birthday-reminder/issues",
+    "homePageUrl": "https://github.com/daskeyboard/daskeyboard-applet--birthday-reminder",
+    "developerRepoUrl": "https://github.com/daskeyboard/daskeyboard-applet--birthday-reminder",
+    "licenseUrl": "http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt",
+    "changelogUrl": "CHANGELOG.md",
+    "readMeUrl": "README.md",
+    "readMeEndUserUrl": "README_ENDUSER.md",
+    "engines": {
+      "das-keyboard-q": "3.1.0"
```

Comment:
Should be the engine of the Das Keyboard Q that you are testing on.
The latest version is 3.2.6